c2cgeoform/templates/grid.jinja2

```diff
@@ -1,14 +1,28 @@
 <div class="list-grid">
 
-  <a class='btn btn-primary btn-new' href="{{request.route_url('c2cgeoform_item', action='edit', id='new')}}">{{_("New")}}</a>
+  <div id="toolbar">
+    <a class="btn btn-primary btn-new"
```

Comment:
btn-new has seems to have been removed in css

Reply:
Yes, I've removed the styles, seems you're suprised I do not removed the class. A class on an element does not meen there is a style on it, it is means it is possible to put one.